nmigen/back/verilog.py

```diff
@@ -16,7 +16,7 @@ class YosysError(Exception):
 def _yosys_version():
     yosys_path = require_tool("yosys")
     version = subprocess.check_output([yosys_path, "-V"], encoding="utf-8")
-    m = re.match(r"^Yosys ([\d.]+)(?:\+(\d+))?", version)
+    m = re.search(r"^Yosys ([\d.]+)(?:\+(\d+))?", version, flags=re.M)
```

Comment:
A comment here would be handy:
```suggestion
    # If Yosys is built with Verific, then Verific license information is printed first.
    m = re.search(r"^Yosys ([\d.]+)(?:\+(\d+))?", version, flags=re.M)
```